# Use-of-try-except-finally

Exception handling is a way of ensuring that your program responds to the error the way we want it, rather than giving your program an error message and stop running in unexpected situations. Error-handling is an important part of Python programming, making your program run reliably without complicating the source code.
